napari/layers/_markers_layer/view/properties.py

```diff
@@ -63,7 +63,8 @@ def __init__(self, layer):
         symbol_comboBox = QComboBox()
         for s in Symbol:
             symbol_comboBox.addItem(str(s))
-        index = self.layer.symbol.value
+        index = symbol_comboBox.findText(
```

Comment:
since you have found a way to get the index without needing to use the enum's value, might you consider setting the enum values to the string forms of their names?

Reply:
That makes sense to me. Will do.